be2-scala/src/security/generateKeys.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
You might want to add some documentation about this (and the rest) in the docs.

Reply:
It is now done !